api_tests/me_get_others.yaml

```
@@ -0,0 +1,72 @@
+# Test generated by RoostGPT for test credential-wallet-api using AI Type Open AI and AI Model gpt-4
+
+
+# RoostTestHash=ae6b8ce5ab
+
+config:
+  target: "{{ $processEnvironment.API_HOST }}"
+  plugins:
+    expect: {}
+  payload:
+    path: "me_get_others.csv"
+    fields: ["field", "value"]
+    skipHeader: true
+  phases:
+    - duration: 60
+      arrivalRate: 20
+
+scenarios:
+  - name: "API Failure Scenarios"
+    flow:
+      - log: "Sending request with invalid payload"
+        post:
+          url: "/me"
+          headers:
+            Authorization: "InvalidToken"
```

```suggestion
            Token: "InvalidToken"
```
APIAuthKey security type has name `Token` not `Authorization`